src/device.c

```diff
@@ -527,14 +531,12 @@ int device_send_config(void)
 	return rc;
 }
 
-void device_publish_data_list(struct l_queue *sensor_id_list)
+void device_publish_data(struct l_queue *sensor_id_list)
```

Comment:
l_quere means queue list?

Reply:
Not. All variables and functions that have the prefix l_ come from the ell library.